### PR TITLE
Update Authorization sidecar image to 1.4.0

### PR DESCRIPTION
**Pull request by bjiang27**
# Description
Update Authorization sidecar to latest 1.4.0 image

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/350 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Comment by randeepsharma** [comment]
Was any tests run on this ?
Pipeline could be used.

**Review by prablr79 — APPROVED** [review]

**Comment by randeepsharma** [comment]
@bjiang27 Please update the codebase with main. gosec should succeed.

**Comment by bjiang27** [comment]
> Was any tests run on this ? Pipeline could be used.

I believe our nightly e2e test run for powerscale tests the nightly driver image with the nightly sidecar image.

**Comment by bjiang27** [comment]
> @bjiang27 Please update the codebase with main. gosec should succeed.

Updated.

**Comment by prablr79** [comment]
@bjiang27 can you update the checklist please ?

**Comment by bjiang27** [comment]
> @bjiang27 can you update the checklist please ?

Updated.

File: service/nodeConnectivityChecker.go [hunk 1]
```diff
@@ -98,7 +98,13 @@ func (s *service) apiRouter(ctx context.Context) {
 	router.HandleFunc(arrayStatus, connectivityStatus).Methods("GET")
 	router.HandleFunc(arrayStatus+"/"+"{arrayId}", getArrayConnectivityStatus).Methods("GET")
 	//start http server to serve requests
-	err := http.ListenAndServe(apiPort, router)
+	server := &http.Server{
```
Comment (randeepsharma) [inline service/nodeConnectivityChecker.go:101]: I dont think this should show up in this PR if branch was rebased with main.

Comment (bjiang27) [reply service/nodeConnectivityChecker.go]: Updated. Rebased with main.

**Pull request closed without merging**